backend/src/main/java/com/pickpick/exception/channel/SubscriptionNotExistException.java

```diff
@@ -5,24 +5,15 @@
 public class SubscriptionNotExistException extends BadRequestException {
 
     private static final String ERROR_CODE = "SUBSCRIPTION_NOT_EXIST";
-    private static final String DEFAULT_MESSAGE = "구독 중이 아닌 채널을 구독 조회";
+    private static final String SERVER_MESSAGE = "구독 중이 아닌 채널을 구독 조회";
     private static final String CLIENT_MESSAGE = "구독 중인 채널이 아니라 취소할 수 없습니다.";
 
     public SubscriptionNotExistException(final Long channelId) {
-        super(String.format("%s -> subscription channel id: %d", DEFAULT_MESSAGE, channelId));
+        super(String.format("%s -> subscription channel id: %d", SERVER_MESSAGE, channelId), CLIENT_MESSAGE,
+                ERROR_CODE);
```

Comment:
아까 줄이 너무 길어지는 것에 대해서 말씀하셨던 것 같아서 제안 드려요
이런 식으로 개선해도 깔끔해보일 것 같네요😄

```suggestion
        String errorMessage = String.format("%s -> subscription channel id: %d", SERVER_MESSAGE, channelId);
        super(errorMessage , CLIENT_MESSAGE, ERROR_CODE);
```

Reply:
너무 좋은 방법이라 해보려고했는데..이친구 생성자 내부에서 super 위에 선언할 수가 없네요 ㅠㅅㅠ 
<img width="696" alt="image" src="https://user-images.githubusercontent.com/55357130/193398449-45fa0a0c-f981-4fa6-8937-c684d55287df.png">
<img width="648" alt="image" src="https://user-images.githubusercontent.com/55357130/193398459-808b2230-d431-404e-a022-2417e5f6157c.png">
추출하면 이렇게 돼요 ㅠㅠ 흑,,

Reply:
😭😭